banzai/images.py

```diff
@@ -215,7 +215,7 @@ def read_images(image_list, pipeline_context):
                 raise dbs.TelescopeMissingException(error_message)
             munge(image, pipeline_context)
             if image.bpm is None:
-                image_utils.load_bpm(image, pipeline_context)
+                image_utils.add_empty_bpm(image)
```

Comment:
Perhaps just add image.bpm = None here? Or perhaps it is already None?

Reply:
It's already `None`, so I've now removed this line altogether.